Issue when starting PyCA

Hello, I've been trying to run PyCA but I having this error when running the start scrip: 

root@ip-10-237-5-67:~/pyCA# ./start.sh
[pyca.config:129:logger_init()] [INFO] Log level set to debug
[pyca.config:89:update_configuration()] [INFO] Configuration loaded from ./etc/pyca.conf
[pyca.utils:78:get_service()] [INFO] Endpoint for org.opencastproject.capture.admin: http://ec2-XX-XXX-XX-XX.us-west-1.compute.amazonaws.com:8080//capture-admin
[pyca.utils:78:get_service()] [INFO] Endpoint for org.opencastproject.ingest: http://ec2-XX-XXX-XX-XX.us-west-1.compute.amazonaws.com:8080//ingest
[pyca.utils:78:get_service()] [INFO] Endpoint for org.opencastproject.scheduler: http://ec2-XX-XXX-XX-XX.us-west-1.compute.amazonaws.com:8080//recordings
[pyca.utils:78:get_service()] [INFO] Endpoint for org.opencastproject.capture.admin: http://ec2-XX-XXX-XX-XX.us-west-1.compute.amazonaws.com:8080//capture-admin
[pyca.utils:78:get_service()] [INFO] Endpoint for org.opencastproject.capture.admin: http://ec2-XX-XXX-XX-XX.us-west-1.compute.amazonaws.com:8080//capture-admin
[pyca.utils:147:register_ca()] [WARNING] Could not set agent statei to offline: (22, 'The requested URL returned error: 405 HTTP method POST is not supported by this URL')
[pyca.schedule:69:get_schedule()] [ERROR] Could not get schedule: (22, 'The requested URL returned error: 404 Not Found')

Then, I cannot see the location in my OC server http://ec2-XX-XXX-XX-XX.us-west-1.compute.amazonaws.com:8080. 

Im using PyCA 2.1 and OC 2.3.0 

Thanks in advance.

--- a/etc/pyca.conf
+++ b/etc/pyca.conf
@@ -124,7 +124,8 @@ command          = 'ffmpeg -nostats -re -f lavfi -r 25 -i testsrc -f lavfi -i si
 
 [server]
 
-# URL of the admin server.
+# Base URL of the admin server. This corresponds to the
+# org.opencastproject.server.url setting of Opencast.
 # Type: string
 # Default: https://octestallinone.virtuos.uos.de
 url              = 'https://octestallinone.virtuos.uos.de'

--- a/pyca/config.py
+++ b/pyca/config.py
@@ -86,6 +86,10 @@ def update_configuration(cfgfile=None):
         raise ValueError('List of files and flavors do not match')
     globals()['__config'] = cfg
     logger_init()
+    if cfg['server'].get('url', '').endswith('/'):
+        logger.warning('Base URL ends with /. This is most likely a '
+                       'configuration error. The URL should contain nothing '
+                       'of the service paths.')
     logger.info('Configuration loaded from %s' % cfgfile)
     check()
     return cfg

